eos/rare-b-decays/b-to-kstar-gamma-bfs2004_TEST.cc

```diff
@@ -156,6 +184,14 @@ class BToKstarGammaBobethCompatibilityTest :
             };
 
             Parameters p = Parameters::Defaults();
+            //p["CKM::abs(V_ub)"] =  0.003631275231633653;
```

Comment:
The test works for me, if I use these values

            p["CKM::abs(V_ub)"] =  0.003631275231633653;
            p["CKM::arg(V_ub)"] = -1.210765774253535;
            p["CKM::abs(V_cb)"] =  0.041996951915501936;
            p["CKM::arg(V_cb)"] =  0.0;
            p["CKM::abs(V_tb)"] =  0.9991111398988599;
            p["CKM::arg(V_tb)"] =  0.0;
            p["CKM::abs(V_us)"] =  0.22534851424454577;
            p["CKM::arg(V_us)"] =  0.0;
            p["CKM::abs(V_cs)"] =  0.9734061890640963;
            p["CKM::arg(V_cs)"] = -0.0000330419933699906;
            p["CKM::abs(V_ts)"] =  0.04121211253368258;
            p["CKM::arg(V_ts)"] = -3.1230250245535283;
            p["CKM::abs(V_td)"] =  0.008859566045351227;
            p["CKM::arg(V_td)"] = -0.38266;

Reply:
I agree, all CKM components need to be fixed